src/components/FormLogin.vue

```diff
@@ -238,10 +253,67 @@ export default defineComponent({
         type="submit"
         color="primary"
         :label="$t('login.form.submitPasswordReset')"
+        @click="formState = 'reset-finished'"
```

Comment:
We need validate email form field value first.

Reply:
Added email validation to both inputs and validation tests in `FormLogin.cy.js`.